dependabot_file.py

```diff
@@ -58,10 +59,23 @@ def build_dependabot_file(
         "terraform": False,
         "github-actions": False,
     }
+    DEFAULT_INDENT = 2  # pylint: disable=invalid-name
 
     if existing_config:
         dependabot_file = existing_config.decoded.decode("utf-8")
+        ecosystem_line = next(
+            line
+            for line in dependabot_file.splitlines()
+            if "- package-ecosystem:" in line
+        )
+        indent = " " * (len(ecosystem_line) - len(ecosystem_line.lstrip()))
+        if len(indent) < DEFAULT_INDENT:
+            print(
+                "Invalid dependabot.yml file. No indentation found. Skipping {repo.full_name}"
+            )
```

Comment:
This should be an f-string :doh:
```suggestion
            print(
                f"Invalid dependabot.yml file. No indentation found. Skipping {repo.full_name}"
            )
```